FogCreek/Glitch-Community
# Superb guide: Handle bad json requests without throwing an error

## Links
https://superb-guide.glitch.me/
https://github.com/FogCreek/Glitch-Community-Work/issues/342

## Changes:
Wrap the body parser handler and catch errors before they go to the normal express/sentry error handler

## How To Test:
Load up the site and run `fetch('/', {method:'POST',cache:'no-cache',headers:{'Content-Type':'application/json'},body:'{asdf}'})` in the console. The network tab should show a 400 error and the project won't log anything
